au.org.access-nri/model/spack/environment/deployment/1-0-0.json

```diff
@@ -0,0 +1,307 @@
+                      "\\w[\\w-]*": {
+                        "type": "array",
+                        "default": [],
+                        "items": {
+                          "type": "string"
+                        }
+                      }
+                    }
+                  },
+                  "variants": {
+                    "oneOf": [
+                      {
+                        "type": "string"
+                      },
+                      {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            "patternProperties": {
+              "(?!^all$)(^\\w[\\w-]*)": {
+                "type": "object",
+                "default": {},
+                "additionalProperties": false,
+                "properties": {
+                  "require": {
+                    "type": "array",
+                    "minItems": 1,
+                    "additionalItems": true,
+                    "items": [
+                        {
+                            "type": "string",
+                            "pattern": "^@[A-Za-z0-9.\\-_]+$"
+                        },
+                        {
+                            "oneOf": [
+                                {
+                                    "type": "object",
+                                    "additionalProperties": false,
+                                    "properties": {
+                                    "one_of": {
+                                        "type": "array",
+                                        "items": {
+                                        "type": "string"
+                                        }
+                                    },
+                                    "any_of": {
+                                        "type": "array",
+                                        "items": {
+                                        "type": "string"
+                                        }
+                                    },
+                                    "spec": {
+                                        "type": "string"
+                                    },
+                                    "message": {
+                                        "type": "string"
+                                    },
+                                    "when": {
+                                        "type": "string"
+                                    }
+                                    }
+                                },
+                                {
+                                    "type": "string"
+                                }
+                            ]
+                        }
+                    ]
+                  },
+                  "version": {
```

Comment:
I'm not sure I understand what this is for? And so I'm unsure where we support it?

Is it the case that we allow some stuff through that we're not explicitly using in the CI/CD?

Reply:
Yeah I'd just left it there - but maybe I should get rid of it for simplicity's sake

Reply:
Got rid of it and reformatted the file